app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/classes/Survey.java

```diff
@@ -1,4 +1,49 @@
 package com.example.coursematchdaddy.clean_architecture_layers.entities.classes;
 
-public class Survey {
```

Comment:
Survey class: In seems like there are some private attributes from the design doc that havent been added here; namely userData, programArea, prerequisites, breadth and distribution. Since theyre private variables, I'll assume they wont have any serious implication for users of Survey. However, you should modify your design doc (especially the constructor of Survey) to avoid any future confusion for people who use your class.

All other classes look good